src/badguy/badguy.hpp

```diff
@@ -230,7 +230,7 @@ class BadGuy : public MovingSprite,
 
   /** Returns true if we might soon fall at least @c height
       pixels. Minimum value for height is 1 pixel */
-  bool might_fall(int height = 1) const;
+  bool might_fall(int height = 1);
```

Comment:
Can you tell me why this method is no longer const? Do we modify the object state with the changes? And if so: Could we prevent that?

Reply:
It changes m_detected_slope. I do this because I want to prevent having two raycasts at a time. So to fix that I alternate between them by checking if m_detected_slope is 0 or not.